scripts/hudson/narayana.sh

```diff
@@ -352,55 +352,23 @@ function build_narayana {
   return 0
 }
 
+# clone WildFly main
+# note that this version does not use 5_BRANCH
+# if we require changes to WildFly sources then we should either
+# a) raise a WildFly PR, or
```

Comment:
Just to be picky, the `or` could be removed

...but feel free to ignore this comment ;-)

Reply:
suitably ignored ;-)

I'll consider it if I ever get time to clean the whole script